core/src/components/refresher/refresher.tsx

```diff
@@ -729,11 +731,35 @@ export class Refresher implements ComponentInterface {
         scrollStyle.transform = backgroundStyle.transform = y > 0 ? `translateY(${y}px) translateZ(0px)` : '';
         scrollStyle.transitionDuration = backgroundStyle.transitionDuration = duration;
         scrollStyle.transitionDelay = backgroundStyle.transitionDelay = delay;
-        scrollStyle.overflow = overflowVisible ? 'hidden' : '';
```

Comment:
One thing I noticed with the MD native refresher is that it sets `--overflow: hidden` (assumes `ion-content`) so the overflow is never set to `hidden` on virtual scroll targets. Not something we need to fix in this PR, but should probably track this.